.github/workflows/validate-build-analyzer.yml

```diff
@@ -0,0 +1,59 @@
+name: Validate Build (analyzer)
+
+on:
+  push:
+    branches:
+      - main
+      - dajusto/run-tests-in-gh-action
```

Comment:
Should we remove this line in all of the yml files before merging?

Reply:
Yes, will do so after obtaining approval, so that the GH can keep on running from this branch